projects/tod_simulator/README.md

```diff
@@ -0,0 +1,79 @@
+# Task Oriented Dialogue (TOD) Script + World
+
```

Comment:
start off with a link to the paper, the abstract, and a bibtex entry.

Reply:
Did in https://github.com/facebookresearch/ParlAI/pull/4201